pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nidaqmx"
-version = "1.1.0"
+version = "1.0.2"
```

Comment:
I don't think we should downgrade mainline. Just double-commit these contributing changes into the release branch

Reply:
Reverted back to 1.1.0. Will have a commit to downgrade to 1.0.2  when cherry pick to release branch.